FromW3resource/src/basic1.py

```diff
@@ -2,9 +2,16 @@
 1. Write a Python program to print the following string in a specific format (see the output). Try to use '\t', instead of ' ', to add large space before some lines.
 
 Twinkle, twinkle, little star,
-	How I wonder what you are! 
+	How I wonder what you are!
 		Up above the world so high,
-		Like a diamond in the sky. 
-Twinkle, twinkle, little star, 
+		Like a diamond in the sky.
+Twinkle, twinkle, little star,
 	How I wonder what you are!
 """
+
+firstline = "Twinkle, twinkle, little star,\n"
+secondline = "\tHow I wonder what you are!\n"
+thirdline = "\t\tUp above the world so high,\n"
+forthline = "\t\tLike a diamond in the sky.\n "
+fifthline = "\tHow I wonder what you are!"
+print(firstline, secondline, thirdline, forthline, firstline, fifthline)
```

Comment:
Have you ever considered that "print(a, b)" will add a space between "a" and "b"?